lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/CommunicationGroupNetworkObserver.cs

```diff
@@ -30,19 +28,21 @@ namespace Org.Apache.REEF.Network.Group.Task.Impl
 {
     /// <summary>
     /// Handles incoming messages sent to this Communication Group.
+    /// Writable version
     /// </summary>
-    public class CommunicationGroupNetworkObserver : ICommunicationGroupNetworkObserver
+    [Obsolete("Need to remove Iwritable and use IstreamingCodec. Please see Jira REEF-295 ", false)]
+    public sealed class CommunicationGroupNetworkObserver : ICommunicationGroupNetworkObserver
```

Comment:
Also later when you replace Iwritable with Istreaming, you might change GeneralGroupCommunicationMessage and the interface that ICommunicationGroupNetworkObserver implements, this class itself would stay with the implementation changes.